app-catalog/src/components/charts/EditorDialog.tsx

```diff
@@ -73,17 +89,22 @@ export function EditorDialog(props: {
   }
 
   useEffect(() => {
-    setChartInstallDescription(`${chart.name} deployment`);
-    fetchChartDetailFromArtifact(chart.name, chart.repository.name).then(response => {
-      if (response.available_versions) {
-        const availableVersions = response.available_versions.map(({ version }) => ({
-          title: version,
-          value: version,
-        }));
-        setVersions(availableVersions);
-        setSelectedVersion(availableVersions[0]);
-      }
-    });
+    if (globalThis.CHART_PROFILE === VANILLA_HELM_REPO) {
+      const versionsArray = AVAILABLE_VERSIONS.get(chart.name);
+      const availableVersions = versionsArray.map(({ version }) => ({ title: version, value: version }));
+      // @ts-ignore
```

Comment:
What is this here for? Can it be removed and the things fixed? Otherwise please leave a justification after it so the reader knows why it's there.


```ts
// @ts-ignore Ignoring this because...
```

Reply:
This has been fixed.